Sources/Load.php

```diff
@@ -849,6 +850,71 @@ function loadUserSettings()
 	call_integration_hook('integrate_user_info');
 }
 
+function loadMinUserSettings($user_id = 0)
+{
+	global $smcFunc, $cache_enable, $modSettings, $language;
+
+	$user_settings_min = array();
+
+	if (empty($userId) || !is_int($userId))
+		return [];
+
+	if (empty($cache_enable) || ($user_settings_min = cache_get_data('user_settings_min-' . $user_id, 120)) == null)
+	{
+		$request = $smcFunc['db_query']('', '
+				SELECT time_offset, additional_groups, id_group, id_post_group, lngfile, smiley_set, time_offset
+				FROM {db_prefix}members
+				WHERE id_member = {int:id_member}
+				LIMIT 1',
+			array(
+				'id_member' => $user_id,
+			)
+		);
+
+		$user_settings_min = $smcFunc['db_fetch_assoc']($request);
```

Comment:
I'm not sure if you don't need some kind of check if the result amount is one.
For exp. line 883 could fail i guess.